pkgs/development/python-modules/notebook_shim/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, jupyter_server
+, pytestCheckHook
+, pytest-tornasync
+}:
+buildPythonPackage rec {
+  pname = "notebook_shim";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jupyter";
+    repo = pname;
```

Comment:
```suggestion
    repo = "notebook_shim";
```

Reply:
Why is it wrong to reuse `pname`? It is a fairly common occurrence.

Or are you suggesting that `pname` should be `notebook-shim`? (which in case is similar to my question about what name to use for the attribute)

Reply:
Re-using pname all over the place does not make code easier to read and I am trying to convince people to avoid it.

> Or are you suggesting that pname should be notebook-shim? 

That, too and then it wouldn't match anyway.